File: proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/NoGLAccountGroupException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class NoGLAccountGroupException extends Exception {
	/**
	 * 
	 */
	private static final long serialVersionUID = 8270078634384672829L;

	public NoGLAccountGroupException(String groupID) {
		super(String
				.format("The G/L account group %s does not exist.", groupID));
	}
}

File: branches/V1.0/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/NoGLAccountGroupException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class NoGLAccountGroupException : Exception
    {
        public NoGLAccountGroupException(String groupID)
            : base(String
                .Format("The G/L account group %s does not exist.", groupID))
        {
        }
    }
}
